elmslie-core/src/main/java/vivid/money/elmslie/core/store/ElmStore.kt

```diff
@@ -32,6 +33,12 @@ class ElmStore<Event : Any, State : Any, Effect : Any, Command : Any>(
 
     private val statesFlow: MutableStateFlow<State> = MutableStateFlow(initialState)
 
+    private val storeListeners: MutableSet<StoreListener<in Event, in State, in Effect, in Command>> =
```

Comment:
Why using 'in'?)

Reply:
why not? )

there is an issue with the generic type StoreListener from GlobalConfig without in